pkg/logs/internal/tailers/file/rotate_nix.go

```diff
@@ -37,8 +38,19 @@ func (t *Tailer) DidRotate() (bool, error) {
 		return true, nil
 	}
 
+	lastReadOffset := t.lastReadOffset.Load()
+	fileOneSize := fi1.Size()
```

Comment:
When we rely on size, it's the same file, there isn't no `fileTwo`. Would it make sense to call it `currentSize`?